protocol/src/imd_interface.c

```diff
@@ -792,6 +792,10 @@ FILE *IIMD_init( const char  *hostname,     imd_int32   *mode,  imd_int32   *IMD
 	{
 		IMDlog = stderr;
 	}
+	else if (strcmp(IMDlogname_, "stdout") == 0)
+    {
```

Comment:
miss one indentation level

Reply:
<img width="796" alt="image" src="https://github.com/LBT-CNRS/MDDriver/assets/34386506/864d3f4d-60b2-43e8-ad63-34498f80b4fd">

I just found that the entire script contains tab characters instead of spaces.

Reply:
no problem for the commit I made before 5b0821616b403252bc25d580c7a28829a31e55f7 and 8fa772f1c969b872fa365c05f7724dd85895b2e7 that imply merges 0d1ad8b8b9533e860b59954df35e989978d2e3aa and 60bceffb96cb085008b9518d38c3379d85f7bb21 respectively